Jenkinsfile

```diff
@@ -4,12 +4,23 @@ import groovy.json.JsonOutput
 @Library('realm-ci') _
 repoName = 'realm-js' // This is a global variable
 
-def nodeVersions = ['8.15.0', '10.15.1']
-def electronVersions = ['2.0.18', '3.0.16', '3.1.8', '4.0.8', '4.1.4', '4.2.6']
+def nodeVersions = ['10.19.0', "12.16.1", "13.0.0"]
+
+//Changing electron versions for testing requires upgrading the spectron dependency in tests/electron/package.json to a specific version. 
+//For more see https://www.npmjs.com/package/spectron 
+def electronVersions = ['5.0.13']
```

Comment:
In fact we should be testing only the latest version of Electron and pushing people to upgrade to latest Electron version as per Electron guidance. 4.2.6 has issues which are fixed in 5 and does not work on our CI if I remember correctly. Realm Studio should be migrated to latest Electron ASAP.

Reply:
I just had a second look at my PR testing the pre-release of this feature and it turns out it was upgrading Electron to v7 (7.1.8) https://github.com/realm/realm-studio/pull/1246. I forgot I already did that upgrade there. We should probably add '7.1.8' that instead of '4.2.6' then :)

Reply:
I think we should move to latest 8.1.1 and switch Realm Studio to use that. I have changed this here to be the lowest possible version in the sense that supporting this we guarantee we support versions from this and above. But it's better not to do that but just test with latest imo

Reply:
As Realm JS has very little integration with the Electron APIs, I'm file with just building (and testing) for the latest major version. Will you make that change?

Reply:
yep lets do this

Reply:
I will do the change